src/SingleArticle.jsx

```diff
@@ -66,12 +64,18 @@ export const SingleArticle = () => {
               <p>{article.body}</p>
               <button>Votes: {article.votes}</button>
               <p>Release Date: {article.created_at}</p>
-              <button onClick={downVote}>
-                <span aria-label="down vote article">down vote: 👎</span>
-              </button>
-              <button onClick={upVote}>
-                <span aria-label="up vote article">up vote: 👍</span>
-              </button>
+              {error ? (
+                <h3>{`Error:${error.status} ${error.data.msg}`}</h3>
```

Comment:
Now the error message is replacing the upvote/downvote buttons. They should display underneath rather than removing the other functionality. As a user, I would probably want to try again rather than have the voting buttons completely disappear on me